dspace-api/src/main/java/org/dspace/submit/lookup/MapConverterModifier.java

```diff
@@ -0,0 +1,115 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.submit.lookup;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+import org.dspace.services.ConfigurationService;
+
+/**
+ * @author Andrea Bollini
+ * @author Kostas Stamatis
+ * @author Luigi Andrea Pascarelli
+ * @author Panagiotis Koutsourakis
+ */
+public class MapConverterModifier {
```

Comment:
Please add JavaDocs to describe this new class.  It may also require some sort of examples, or extra details in the `init()` method, as there are currently no inline comments.

This class also appears to have no tests.  Is there some way to test the logic in the `init()` method?